### PR TITLE
feat: add initial support for app attestation

### PR DESCRIPTION
**Pull request by jleach**
The third and final component to the application attestation mechanics is the logic in the BC Wallet to bridge the gap between the RN module and the attestation controller (via traction).

This PR adds support to interface with the [RN module @hyperledger/aries-react-native-attestation](https://www.npmjs.com/package/@hyperledger/aries-react-native-attestation) in the Bifold repo and use it to communicate over DIDComm basic messages to a controller that supports attestation.

### CODE REVIEW

**Comment by jleach** [comment]
Converted back to draft. Few conflicts to fix first.

File: app/ios/Podfile.lock [hunk 1]
```diff
@@ -362,6 +362,8 @@ PODS:
   - React-jsinspector (0.66.5)
   - React-logger (0.66.5):
     - glog
+  - "react-native-attestation (1.0.0-alpha.99+1f2c467)":
```
Comment (bryce-mcmath) [inline app/ios/Podfile.lock:448]: What is this `+hash` at the end of the version?

**Review by bryce-mcmath — DISMISSED** [review]
Clean! LGTM

File: app/ios/Podfile.lock (hunk 1 above)
Comment (jleach) [reply app/ios/Podfile.lock:448]: Just something added by yarn so I'll leave it.

Comment (cvarjao) [reply app/ios/Podfile.lock:448]: It is actually a lerna thing:
https://github.com/lerna/lerna/tree/main/libs/commands/publish#--canary

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**